pkg/operator/status.go

```diff
@@ -207,6 +207,9 @@ func (optr *Operator) initializeClusterOperator() (*configv1.ClusterOperator, er
 	// RelatedObjects are consumed by https://github.com/openshift/must-gather
 	co.Status.RelatedObjects = []configv1.ObjectReference{
 		{Resource: "namespaces", Name: "openshift-machine-config-operator"},
+		{Group: "machineconfiguration.openshift.io", Resource: "machineconfigpools", Name: "master"},
+		{Group: "machineconfiguration.openshift.io", Resource: "machineconfigpools", Name: "worker"},
```

Comment:
We also need to support custom pools right?

Reply:
I've already a follow up to dynamically collect and populate this - I'm running through some testing and will open it once ready

Reply:
OK, so land as is and then improve later?  Cool by me!

Reply:
yup, that's the plan, at the beginning I actually just thought about having `master` only here. The information we gather from this is questionable also, logs from controller is what we'll look at anyway (and we gather that)